lib/server/openload.py

```diff
@@ -21,17 +21,74 @@
 # */
 import re
 import util
+import urllib2
+from aadecode import AADecoder
 
-__author__ = 'Lubomir Kucera'
+__author__ = 'Jose Riha/Lubomir Kucera'
 __name__ = 'openload'
 
 
 def supports(url):
-    return re.search(r'openload\.io/embed/.+', url) is not None
+    return re.search(r'openload\.\w+/embed/.+', url) is not None
 
 
 def resolve(url):
-    source = re.search(r'<source[^>]+?src=\"([^\"]+)\"', util.request(url), flags=re.S)
-    if source:
-        return [{'url': source.group(1)}]
-    return None
+    # uses code fragments from https://github.com/LordVenom/venom-xbmc-addons/blob/master/plugin.video.vstream/resources/hosters/openload.py
```

Comment:
This line is too long. I'd recommend trimming it to `# Uses code fragments from https://github.com/LordVenom/venom-xbmc-addons` and placing it above `def resolve(url):`.